### PR TITLE
Add base unit Tests

### PR DESCRIPTION
**Pull request by BryanHo10**

### CODE REVIEW

File: USFMToolsSharp.Renderers.Docx.Tests/DocxRendererTests.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            Assert.AreEqual("1", renderDoc("\\c 1").Paragraphs[0].Text);
+            Assert.AreEqual("-1", renderDoc("\\c -1").Paragraphs[0].Text);
+            Assert.AreEqual("0", renderDoc("\\c 0").Paragraphs[0].Text);
+        }
+        [TestMethod]
+        public void TestVerseRender()
+        {
+            Assert.AreEqual("1This is a simple verse.", renderDoc("\\c 1 \\v 1 This is a simple verse.").Paragraphs[1].ParagraphText);
+            Assert.AreEqual("1This is a simple verse.2verse 2", renderDoc("\\c 1 \\v 1 This is a simple verse. \\v 2 verse 2").Paragraphs[1].ParagraphText);
+            Assert.AreEqual("2verse 2", renderDoc("\\c 1 \\v 1 This is a simple verse. \\c 2 \\v 2 verse 2").Paragraphs[3].ParagraphText);
```
Comment (rbnswartz) [inline USFMToolsSharp.Renderers.Docx.Tests/DocxRendererTests.cs]: This looks weird. 2verse 2?

File: USFMToolsSharp.Renderers.Docx.Tests/DocxRendererTests.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        public void TestFootnoteRender()
+        {
+            Assert.AreEqual("Footnotes", renderDoc("\\c 1 \\v 1 This is a simple verse. \\f + \\ft Hello Friend \\f*").Paragraphs[2].ParagraphText);
+            Assert.AreEqual("1Hello Friend", renderDoc("\\c 1 \\v 1 This is a simple verse. \\f + \\ft Hello Friend \\f*").Paragraphs[3].ParagraphText);
+            Assert.AreEqual("1Hello Fried Friend", renderDoc("\\c 1 \\v 1 This is a simple verse. \\f + \\ft \\fqa Hello Fried Friend \\f*").Paragraphs[3].ParagraphText);
+        }
+
+        public XWPFDocument renderDoc(string usfm)
+        {
+            render.clearDocumentElements();
```
Comment (rbnswartz) [inline USFMToolsSharp.Renderers.Docx.Tests/DocxRendererTests.cs]: You should actually just create a whole new document. When unit testing you want to start with a clean slate. This doesn't guarantee that the slate is clean

File: USFMToolsSharp.Renderers.Docx/DocxRenderer.cs [hunk 3]
```diff
@@ -287,7 +287,13 @@ public void createBookHeaders(string bookname)
 
             bookNameCount++;
         }
-
+        public void clearDocumentElements()
```
Comment (rbnswartz) [inline USFMToolsSharp.Renderers.Docx/DocxRenderer.cs]: I don't think this is necessary if you make the change to the tests that I recommended

File: USFMToolsSharp.Renderers.Docx.Tests/DocxRendererTests.cs (hunk 2 above)
Comment (BryanHo10) [reply USFMToolsSharp.Renderers.Docx.Tests/DocxRendererTests.cs]: @rbnswartz Yeah you're right. I updated it

File: USFMToolsSharp.Renderers.Docx.Tests/DocxRendererTests.cs (hunk 1 above)
Comment (BryanHo10) [reply USFMToolsSharp.Renderers.Docx.Tests/DocxRendererTests.cs]: I'll change the test usfm

**Pull request merged**